anaconda.spec.in

```diff
@@ -184,6 +184,7 @@ Requires: tmux
 # install time crash handling
 Requires: gdb
 Requires: rsync
+Requires: zram-generator-defaults
```

Comment:
Do we want to have this dependency in `anaconda-install-env-deps`? Or should it be defined in the `lorax` templates and `spin-kickstarts` kickstart files?

Reply:
Hmm, if we will deprecate the ``inst.zram`` boot option then I don't see any reason to have it as Anaconda dependency.

Reply:
Hmm, if we will deprecate the inst.zram boot option then I don't see any reason to have it as Anaconda dependency.

Reply:
Changed to `Recommends`.